### PR TITLE
Lf 101890940 new date format

### PR DESCRIPTION
**Pull request by lfender6445**
see notes here: https://github.com/rentpath/service_moving_form/pull/38

### CODE REVIEW

File: dist/jquery-moving-service.js [hunk 1]
```diff
@@ -199,7 +199,7 @@
               $("#moving_lead_MovingDate").datepicker({
                 minDate: "+2w",
                 maxDate: "+6m",
-                dateFormat: "mm/dd/yy"
+                dateFormat: "mm-dd-yy"
```
Comment (pasha) [inline dist/jquery-moving-service.js]: Just to note, clients may need to update format of the label on a Form.

**Comment by pasha** [comment]
:+1:

File: dist/jquery-moving-service.js (hunk 1 above)
Comment (lfender6445) [inline dist/jquery-moving-service.js]: form and labels html comes from service

**Comment by pasha** [comment]
@abouius are you good with a change?

**Comment by abouius** [comment]
Yep LGTM :+1:

**Pull request merged**